src/translations/sqlb_ar_SA.ts

```diff
@@ -2756,10 +2837,10 @@ You can drag SQL statements from the Schema column and drop them into the SQL ed
         <translation>يُغلق هذا الزر الاتصال بملف قاعدة البيانات المفتوح حاليًا</translation>
     </message>
     <message>
-        <location filename="../MainWindow.cpp" line="218"/>
-        <location filename="../MainWindow.cpp" line="3347"/>
+        <location filename="../MainWindow.cpp" line="227"/>
+        <location filename="../MainWindow.cpp" line="3429"/>
         <source>Ctrl+W</source>
-        <translation></translation>
+        <translation>Ctrl+W</translation>
```

Comment:
Shortcuts should not be translated.
See the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts) for further information.